openedx/core/djangoapps/theming/models.py

```diff
@@ -0,0 +1,16 @@
+"""
+Comprehensive Theme related models.
+"""
+from django.db import models
+from django.contrib.sites.models import Site
+
+
+class SiteTheme(models.Model):
+    """
+    This is where the information about the site's theme gets stored to the db.
```

Comment:
Let's add some specific information to the docstring about each of these fields.